banzai/photometry.py

```diff
@@ -241,3 +243,39 @@ def do_stage(self, image):
         except Exception:
             logger.error(logs.format_exception(), image=image)
         return image
+
+
+class PhotometricCalibrator(Stage):
+    color_to_fit = {'gp': 'g-r', 'rp': 'r-i', 'ip': 'r-i', 'zs': 'i-z'}
+
+    def __init__(self, runtime_context):
+        super(PhotometricCalibrator, self).__init__(runtime_context)
+
+    def do_stage(self, image):
+        if image.filter not in ['gp', 'rp', 'ip', 'zs']:
```

Comment:
May help to add a log here: "Image taken in a filter that will not be photometrically calibrated" or something?

Reply:
There should be some documentation on why the color to be fitted is dependent on the filter.  it is a design choice, but since we are fitting against a catalog color, not instrumental colors were you are limited to what you observed, there is a real choice to me made.  The PanSTARRS to SDSS conversion is entirely based on g-i I think, and they did that fro a good reason.

Reply:
This is definitely a design choice. The primary reason is so we can estimate useful color terms for non stellar SEDs. As we do not know the user's target of interest and are not generating all sky catalogs, I would argue that we should use closer colors. Open to discussion though.